docs/blog/2019-04-19-gatsby-why-we-write/index.md

```diff
@@ -0,0 +1,123 @@
+
+People from each of the communities who have discovered Gatsby are the experts on delivering the message of "why Gatsby" to that community.
+
+Our objective is **covering the ground** — or if you prefer, _breadth-first search_ rather than _depth-first search_ — making sure that each meaningful message has at least some basic material (both documentation of _how_ and the _why_ of case studies and success stories).
+
+In our content database, which lives in Airtable, we store an auto-updating column in the Messages table, which tracks the # of content pieces per message.
+
+But...there's a lot of ground to cover? **Yes, there is.**
+
+It's essential to our mission, though, because in order to become the default choice for building on the web, people need to realize that _Gatsby can do anything you conceivably want_ - this is why Wordpress became 30% of the internet! - and we emphasize that by writing about all use cases.
+
+_"Gatsby can't do X"_ gets rebutted by _"Here's a case study of Gatsby doing X"_.
+
+(For more background on this concept, read about ["economies of scope"](https://www.ribbonfarm.com/2012/10/15/economies-of-scale-economies-of-scope/) or think about companies that have flourished by providing access to literally everything their customers needed — the 1897 Sears Roebuck catalogue with [73K items](https://wpengine.com/reduce-reuse-refresh/); today, Amazon (the "Everything Store") with [100s of millions of items](https://wpengine.com/reduce-reuse-refresh/).)
+
+The challenge, is of course _how do we write all of that content?_
+
+### Principle 4: Leverage ourselves through collaboration.
+
+Some examples:
+
+- As an Inkteam, we're prioritizing podcasts to conference talks, because they take way less time and build on an existing community/listener base. The exception is key events like JAMStack and DrupalCon that are landmark events for specific communities.
+
+- We encouraging the community to give conference talks, write on their own blogs, and on the Gatsby blog. Individuals can establish themselves publicly as talented developers and Gatsby experts, while companies using Gatsby
+
+- Jason's livestreams are fundamentally collaborative — inviting one person who is an expert in a different technology for cross-pollination.
+
+- We maintain close relationships with educators who create comprehensive, complemenrary content, and provide encouragement to first-time users that _you can do this!_.
+
+If you're doing interesting, innovative things with Gatsby, especially in a professional context, we'd love you to write blog posts, tutorials, and documentation about what you're doing!
+
+### Principle 5: Content is not just individual pieces of work, it's compilations.
+
+If content were music, it isn't just "songs" -- it's "playlists", "albums", "discographies", "soundtracks", and "multi-box sets".
```

Comment:
```suggestion
If content were music, it wouldn't consist of just "songs" -- it would also include "playlists", "albums", "discographies", "soundtracks", and "multi-box sets".
```